solar_system.rb

```diff
@@ -0,0 +1,41 @@
+class SolarSystem
+  attr_reader :star_name, :planet
+
+  def initialize(star_name)
+    @star_name = star_name
+    @planet = []
+  end
+
+  def add_planet(instance_of_planet)
+    @planet << instance_of_planet
+  end
+
+  def list_planets
+    planet_list = "Planets orbiting #{@star_name}:\n"
+    @planet.each_with_index do |planet, index|
+      planet_list << "#{index + 1}. #{planet.name}\n"
+    end
+    return planet_list
+  end
+
+  def find_planet_by_name(planet)
+    selected_planet = @planet.select { |other_planet| other_planet.name.downcase == planet.downcase }
+    selected_planet.each { |each_planet| puts each_planet.summary }
+  end
+
+  def make_new_planet
+    puts "What is the name of your planet?"
+    user_planet = gets.chomp.capitalize
+    puts "What color is the planet?"
```

Comment:
It might make sense to put this method that interacts with the user in `main.rb` instead of `SolarSystem`.